statistics/table.go

```diff
@@ -465,18 +465,35 @@ func outOfRangeEQSelectivity(ndv, modifyRows, totalRows int64) float64 {
 }
 
 // getEqualCondSelectivity gets the selectivity of the equal conditions.
-func (coll *HistColl) getEqualCondSelectivity(idx *Index, bytes []byte, usedColsLen int) float64 {
+func (coll *HistColl) getEqualCondSelectivity(sc *stmtctx.StatementContext, idx *Index, bytes []byte, usedColsLen int, colRanges []*ranger.Range) (float64, error) {
```

Comment:
It's better to use `idxPointRange *ranger.Range` instead of `colRanges []*ranger.Range`.